packages/manager/apps/web/client/app/dns-zone/zone/dns-zone.service.js

```diff
@@ -0,0 +1,12 @@
+export default class DNSZoneService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  getCapabilities(zoneName) {
+    return this.$http
```

Comment:
no cache required?

Reply:
I've only moved this file, no changes made on files other than routing/module declaration